src/spiritual/spiritual-gui/gui@wunderbyte.com/polyfills.js

```diff
@@ -88,6 +88,24 @@
 					}
 				}
 				return undefined;
+			},
+			diffSymmetric: function diffSymmetric(otherArr) {
```

Comment:
Although perhaps not a polyfill, I guess, but yeah.

Reply:
where would you put it?